runtime/v1/shim/service.go

```diff
@@ -547,13 +547,16 @@ func (s *Service) checkProcesses(e runc.Exit) {
 	}
 }
 
-func shouldKillAllOnExit(bundlePath string) (bool, error) {
+func shouldKillAllOnExit(bundlePath string, ctx context.Context) (bool, error) {
```

Comment:
unneeded change

Reply:
this is for passing Context to log.G(ctx) call inside shouldKillAllOnExit